src/storage/mvcc/reader/scanner/forward.rs

```diff
@@ -270,6 +271,7 @@ impl<S: Snapshot, P: ScanPolicy<S>> ForwardScanner<S, P> {
                         return Ok(Some(output));
                     }
                     HandleRes::Skip(key) => key,
+                    HandleRes::MoveToNext(_) => continue,
```

Comment:
Do we need to handle the move next result for backford scanner?

Reply:
No. The implementation of backward scanner is different from forward scanner which abstracts a trait for different scan policies. Backward scanner handles it and `test_scan_access_locks` covers it.

https://github.com/tikv/tikv/blob/c04fd41029c28f54e934830f63424f0aa34e2654/src/storage/mvcc/reader/scanner/backward.rs#L177